docs/collecting-data/collecting-from-own-applications/javascript-trackers/browser-tracker/browser-tracker-v3-reference/plugins/vimeo-tracking/index.md

```diff
@@ -0,0 +1,207 @@
+---
+title: 'Vimeo Tracking'
+date: '2022-01-12'
+sidebar_position: 17000
+---
+
+```mdx-code-block
+import Block5966 from '@site/docs/reusable/javascript-tracker-release-badge-v3/_index.md'
```

Comment:
Could you give this a meaningful name please? Like `import WhatThisIsAbout from ...`